ddtrace/settings/config.py

```diff
@@ -248,16 +248,6 @@ def source(self) -> _ConfigSource:
             return "env_var"
         return "default"
 
-    def _telemetry_value(self) -> str:
-        val = self.value()
-        if val is None:
-            return ""
-        elif isinstance(val, (bool, int, float)):
-            return str(val).lower()
-        elif isinstance(val, dict):
-            return ",".join(":".join((k, str(v))) for k, v in val.items())
-        return str(val)
```

Comment:
So I saw:
> Avoids reporting integers, floats, booleans and None types as strings in instrumentation telemetry payloads

Not against this change, but was there a reason we did this initially instead of using the original types? Did something change to prompt this?

Reply:
Java currently reports Null values as empty strings and stringify booleans (current behavior) while golang/dotnet/nodejs/ruby report nil values and booleans in their expected json representation. Here I am aligning with how the majority of languages handle telemetry values. 

I have no idea why Java was implemented in this way.